frontend/src/App.scss

```diff
@@ -84,6 +87,28 @@ button {
       img {
         height: 10rem;
       }
+      .rightDirection {
+        position: absolute;
+        top: 13rem;
+        right: 2rem;
+        font-size: 2rem;
+      }
+      .leftDirection {
+        position: absolute;
+        top: 13rem;
+        left: 2rem;
+        font-size: 2rem;
+      }
```

Comment:
your menu should have a `position: relative` to become the reference for the absolute positions of its children. 

prefer % for absolute positions : this will be more robust than font size (imagine if someone is zooming on the text : all your positions will break)